src/main/java/org/jabref/gui/entryeditor/RelatedArticlesTab.java

```diff
@@ -141,26 +130,20 @@ private ScrollPane getPrivacyDialog(BibEntry entry) {
         line1.setWrappingWidth(1300.0);
         Text line2 = new Text(Localization.lang("This setting may be changed in preferences at any time."));
         Hyperlink mdlLink = new Hyperlink(Localization.lang("Further information about Mr DLib. for JabRef users."));
-        mdlLink.setOnAction(new EventHandler<ActionEvent>() {
-            @Override
-            public void handle(ActionEvent event) {
-                try {
-                    JabRefDesktop.openBrowser("http://mr-dlib.org/information-for-users/information-about-mr-dlib-for-jabref-users/");
-                } catch (IOException e) {
-                    LOGGER.error("Error opening the browser to Mr. DLib information page.", e);
-                    dialogService.showErrorDialogAndWait(e);
-                }
+        mdlLink.setOnAction(event -> {
+            try {
+                JabRefDesktop.openBrowser("http://mr-dlib.org/information-for-users/information-about-mr-dlib-for-jabref-users/");
+            } catch (IOException e) {
+                LOGGER.error("Error opening the browser to Mr. DLib information page.", e);
+                dialogService.showErrorDialogAndWait(e);
             }
         });
 
-        button.setOnAction(new EventHandler<ActionEvent>() {
-            @Override
-            public void handle(ActionEvent event) {
-                JabRefPreferences prefs = JabRefPreferences.getInstance();
-                prefs.putBoolean(JabRefPreferences.ACCEPT_RECOMMENDATIONS, true);
-                dialogService.showWarningDialogAndWait(Localization.lang("Restart"), Localization.lang("Please restart JabRef for preferences to take effect."));
-                setContent(getRelatedArticlesPane(entry));
-            }
+        button.setOnAction(event -> {
+            JabRefPreferences prefs = JabRefPreferences.getInstance();
```

Comment:
Please use the EntryEditorPreferences which are already passed via constructor. Apart form that lgtm

Reply:
But shouldn't this setting not go into the `JabRefPreferences` and not the preferences of the EntryEditor? When I see this correctly in the above screenshot, this setting was supposed to be available main preferences.